admin/starter/tests/_support/SessionTestCase.php

```diff
@@ -7,10 +7,7 @@
 use CodeIgniter\Test\CIUnitTestCase;
 use CodeIgniter\Test\Mock\MockSession;
 
-/**
- * @internal
- */
-final class SessionTestCase extends CIUnitTestCase
+abstract class SessionTestCase extends CIUnitTestCase
```

Comment:
It seems SessionTestCase is not needed any more. Because `mockSession()` is in `CIUnitTestCase`.
https://github.com/codeigniter4/CodeIgniter4/blob/9c07842adda5a9196e6203647d6d25cb793f512a/system/Test/CIUnitTestCase.php#L328

Reply:
I agree, let's delete this file. See my other comment.